### PR TITLE
feat: support configrate the jobservice logger sweeper duration

### LINKED ISSUE #10958: Make jobservice's sweeper duration configurable
**Is your feature request related to a problem? Please describe.**
I wanted to check tag retention job logs but found out that they are available only for 1 day as it's hardcoded in jobservice [configuration](https://github.com/goharbor/harbor/blob/v1.10.1/make/photon/prepare/templates/jobservice/config.yml.jinja)
Even in harbor-helm it's set to 14 days which is more applicable.
I understand I can change it manually after running prepare script, but I would like it to be configurable option in harbor.yml

**Describe the solution you'd like**
Add jobservice.sweeper_duration parameter to harbor.yml

### PR DESCRIPTION
**Pull request by chlins**
Support configurate the jobservice logger sweeper duration from harbor.yml.

Closes: #10958

Thank you for contributing to Harbor!

# Comprehensive Summary of your change

# Issue being fixed
Fixes #10958 

Please indicate you've done the following:
- [x] Well Written Title and Summary of the PR
- [x] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [x] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

**Review by wy65701436 — APPROVED** [review]
lgtm

**Comment by MinerYang** [comment]
If we do the prepare migration script for v2.8 all in this PR,  following items need to be take cared in the jinja file as well:
1. `chart.absolute_url` https://github.com/goharbor/harbor/blob/6113469a567623054a58d0f5506b90a55adca7ff/make/harbor.yml.tmpl#L120
2. `external_redis.chartmuseum_db_index: 3` https://github.com/goharbor/harbor/blob/6113469a567623054a58d0f5506b90a55adca7ff/make/harbor.yml.tmpl#L190
3.  `external_redis.username`

File: make/photon/prepare/migrations/version_2_8_0/harbor.yml.jinja [hunk 1]
```diff
@@ -0,0 +1,574 @@
...
+  #  <host_sentinel1>:<port_sentinel1>,<host_sentinel2>:<port_sentinel2>,<host_sentinel3>:<port_sentinel3>
+  host: {{ external_redis.host }}
+  password: {{ external_redis.password }}
+  # Redis AUTH command was extended in Redis 6, it is possible to use it in the two-arguments AUTH <username> <password> form.
+  {% if external_redis.username is defined %}
+  username: {{ external_redis.username }}
+  {% else %}
+  # username:
+  {% endif %}
+  # username:
```
Comment (MinerYang) [inline make/photon/prepare/migrations/version_2_8_0/harbor.yml.jinja]: Seems like this line is needless

**Review by MinerYang — APPROVED** [review]
lgtm

**Pull request merged**